src/data/polynomial/derivative.lean

```diff
@@ -214,5 +214,31 @@ le_antisymm
 
 end domain
 
+section char_zero
```

Comment:
Is there a reason for this lemma to be in a new section instead of the previous one?